src/controllers/JobRoleController.ts

```diff
@@ -1,12 +1,12 @@
 import express from "express"
```

Comment:
There are missing tests for JobRoleController.

Reply:
Added